README.md

```diff
@@ -44,7 +44,8 @@ Market maker has several configurable parameters for creating markets. These par
 - `num_markets`: Number of markets an agent to allowed to create before, default is 1
 - `market_fee`: Fees for creating a market, default is 2 unit (Eth, xDAI, etc...)
 - `initial_funds`: Initial funds for the market, default is 1 unit (Eth, xDAI, WxDAI, etc...)
-- `market_timeout`: Time for which the market should be active after opening, default is 7 days
+- `market_timeout`: Maximum time allowed for finalizing answer on `realitio`, default is 7 days
```

Comment:
Why 7 days? I think that's a lot. It should be 1 day.

Reply:
It's configurable

Reply:
Please change this to 1 day by default in all referenced components and env variables.